llvm/test/CodeGen/AArch64/extract-vector-elt.ll

```diff
@@ -143,11 +143,12 @@ define i64 @extract_v2i64_extract_build_vector_opaque(<2 x i64> %a, i32 %c) {
 ; CHECK-SD:       // %bb.0: // %entry
 ; CHECK-SD-NEXT:    sub sp, sp, #16
 ; CHECK-SD-NEXT:    .cfi_def_cfa_offset 16
-; CHECK-SD-NEXT:    adrp x8, .LCPI8_0
+; CHECK-SD-NEXT:    mov x8, #-31 // =0xffffffffffffffe1
```

Comment:
Degrading test coverage in the non-SVE case?

Reply:
This PR adds test-cases with scalable vectors, which are only supported *with* SVE.

Reply:
Yes, but it is mixing tests with fixed vectors. Split the test for the scalable cases?

Reply:
Do you want to move the new tests to a different file?